packages/core/src/generatePolicy.js

```diff
@@ -246,16 +316,28 @@ function createModuleInspector(opts = {}) {
       packageToBuiltinImports.set(packageName, [])
     }
     let packageBuiltins = packageToBuiltinImports.get(packageName)
-    packageBuiltins = [...packageBuiltins, ...moduleBuiltins]
+    packageBuiltins = [...(packageBuiltins ?? []), ...moduleBuiltins]
     packageToBuiltinImports.set(packageName, packageBuiltins)
   }
 
-  function generatePolicy({ policyOverride, includeDebugInfo } = {}) {
+  /**
+   * @type {GeneratePolicyFn}
+   */
+  function generatePolicy({ policyOverride, includeDebugInfo = false }) {
+    /** @type {import('./schema').Resources} */
```

Comment:
Yes, it could be used here without worrying about an export (I think!), but if `Resources` becomes generic in the future, we would need to re-implement those  type args in `@template` tags.  That said, you could probably get away with it here--kind of like you can get away with writing JS without semicolons.

_Personally,_ I don't want to think too hard about whether or not it's safe to create the `@typedef` reference, so I choose not to and make that tradeoff.  Similarly, I don't like thinking about whether or not ASI is going to work either, so I always use semicolons. :smile:

Reply:
As the best practices around this are far from set in stone, I'm trying to explore options in case we come up with something that's more convenient. 

Also note that making this pattern more popular makes SES slower, because it relies on a transform to replace each `import(` with something else

Reply:
The transform in question doesn't happen in SES, fwiw.